src/Microsoft.Health.Fhir.Shared.Core.UnitTests/Features/Search/SearchParameters/SearchParameterFixtureData.cs

```diff
@@ -66,8 +68,17 @@ private static async Task<FhirTypedElementToSearchValueConverterManager> CreateF
             var codeSystemResolver = new CodeSystemResolver(ModelInfoProvider.Instance);
             await codeSystemResolver.StartAsync(CancellationToken.None);
 
-            var fhirElementToSearchValueConverters =
-                types.Select(x => (ITypedElementToSearchValueConverter)Mock.TypeWithArguments(x, referenceSearchValueParser, codeSystemResolver));
+            var fhirElementToSearchValueConverters = new List<ITypedElementToSearchValueConverter>();
+
+            foreach (Type type in types)
+            {
+                // The extension converters will be added to the converter dictionary by the converter manager
+                if (!type.Name.Contains("ExtensionConverter"))
```

Comment:
>type.Name.Contains("ExtensionConverter") [](http://example.com/codeflow?start=21&length=40)

I would try to use `nameof`
another option is ExtensionsConverter is a generic type, while all other converters are not. 
`if (type.ContainsGenericParameters)`

Reply:
Changed to use `nameof` in 51adeb6.